apis/beacon/light_client/sync_committee_roots.yaml

```diff
@@ -0,0 +1,67 @@
+get:
+  operationId: getSyncCommitteeRootsByRange
+  summary: Get `SyncCommittee` SSZ HashTreeRoot in a requested sync committee period range
+  description: |
+    Requests the [`SyncCommittee`](https://github.com/ethereum/consensus-specs/blob/v1.2.0/specs/altair/beacon-chain.md#synccommittee) SSZ HashTreeRoot in the sync committee period range `[start_period, start_period + count)`.
+  tags:
+    - Beacon
+  parameters:
+    - name: start_period
+      description: "The start sync committee period we want to get SSZ HashTreeRoot from"
+      in: query
+      required: true
+      schema:
+        $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+    - name: count
+      description: "Number of sync committee periods we want to get SSZ HashTreeRoots starting from start_period"
+      in: query
+      required: true
+      schema:
+        $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+  responses:
+    "200":
+      description: Success
+      content:
+        application/json:
+          schema:
+            title: getSyncCommitteeRootsByRangeResponse
+            type: object
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Root'
+    "400":
+      description: Malformed or missing request parameter
+      content:
+        application/json:
+          schema:
+            $ref: "../../../beacon-node-oapi.yaml#/components/schemas/ErrorMessage"
+          examples:
+            InvalidSyncPeriod:
+              value:
+                code: 400
+                message: "Invalid sync committee period requested"
+            InvalidCount:
+              value:
+                code: 400
+                message: "Invalid count requested"
+            MissingStartPeriodValue:
+              value:
+                code: 400
+                message: "Missing `start_period` value"
+            MissingCountValue:
+              value:
+                code: 400
+                message: "Missing `count` value"
+    "404":
+      description: "One of periods not found"
```

Comment:
`One or more periods could not be found`

Reply:
404 isn't a great code though, its the same as if you hit a route thats not implemented...